serenata_toolbox/speeches.py

```diff
@@ -0,0 +1,88 @@
+import os
+import pandas as pd
+import numpy as np
+import xml.etree.ElementTree as ET
+import urllib
+
+from datetime import datetime
+
+class Speeches:
+
+    URL = 'http://www.camara.leg.br/SitCamaraWS/SessoesReunioes.asmx/ListarDiscursosPlenario?dataIni={dataIni}&dataFim={dataFim}&codigoSessao=&parteNomeParlamentar=&siglaPartido=&siglaUF='
```

Comment:
For readability sake you can use [a trick](https://docs.python.org/3/reference/lexical_analysis.html#string-literal-concatenation) when the string is too long: parenthesis without commas (otherwise you break it into a tuple). This like would become something like (and the value of the string would be exactly the same)

```python
    URL = (
        'http://www.camara.leg.br/SitCamaraWS/SessoesReunioes.asmx/ListarDiscursosPlenario'
        '?dataIni={dataIni}'
        '&dataFim={dataFim}'
        '&codigoSessao=&parteNomeParlamentar=&siglaPartido=&siglaUF='
    )
```